src/lib/chips/chip-list.ts

```diff
@@ -379,6 +379,12 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
       this._updateFocusForDestroyedChips();
 
       this.stateChanges.next();
+
+      // Sync the `selectable` state to all the chips. Needs to happen
+      // on the next tick to avoid `changed after checked` errors.
+      Promise.resolve().then(() => {
```

Comment:
Is there a way we could do this without adding a new async task? This usually breaks any unit tests that use the component and `fakeAsync`

Reply:
Not without reworking a bunch of logic. The problem is that this state depends on something that comes through a `QueryList` which will always trigger a "changed after checked". I don't see this causing any G3 sync issues since it never really worked to begin with so people probably don't have tests for it.